cordlib/src/main/kotlin/com.luxoft.blockchainlab.corda.hyperledger.indy/service/ConnectionService.kt

```diff
@@ -52,7 +51,7 @@ class ConnectionService (services: AppServiceHub) : SingletonSerializeAsToken()
             agentLogin ?: throw RuntimeException("Agent websocket endpoint specified but agent user name is missing")
             agentPassword ?: throw RuntimeException("Agent websocket endpoint specified but agent password is missing")
 
-            PythonRefAgentConnection().apply { connect(agentWsEndpoint, agentLogin, agentPassword).awaitFiber() }
+            PythonRefAgentConnection().apply { connect(agentWsEndpoint, agentLogin, agentPassword).toBlocking().value() }
```

Comment:
Why do we move away from "awairFiber" extension?

Reply:
Because there is no fiber context, and they won`t work and will throw NPE...